### DISCUSSION TITLE
guide/markdown/advance/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#175] [docs-comment]
# guide/markdown/advance/

卡片 对于想要突出显示的内容，可以将其放在 卡片容器 ::: card 中。 如果需要在空间足够时并排显示多个卡片，可以使用 card-grid 容器，包裹 多个卡片。 语法 卡片 支持 可选的 title 和 icon。 其中， icon 支持传入 图片链接，也可以传入 iconify 图标名。 示例 输入： 输出： 输入： 输出： 相关信息 如果你...

https://theme-plume.vuejs.press/guide/markdown/advance/

**Comment by cjhalw** [comment]
^1.0.0-rc.98版本文件树添加.md后缀的文件路径时会提示：

**Reply by cjhalw** [reply]
Internal server error: Element is missing end tag.<a href="http://README.md" target="_blank" rel="noopener noreferrer">

**Reply by pengzhanbo** [reply]
我没有重现出该问题，但是发现了当仅输入 `-` 符号而不写任何文件名时有该错误，这个问题已排查到原因，下个版本修复。
